### PR TITLE
Update UI to use new resource color palette

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Closes [RHOAIENG-14753](https://issues.redhat.com/browse/RHOAIENG-14753)

## Description
The color palette used for resources has been updated in the new [guidelines](https://docs.google.com/document/d/1OdaE6LjTikTMTIOHvgGQzRDJpm9Bn-kPxsfoqRh5tsU/edit#heading=h.v94u92bpa671)

## How Has This Been Tested?
Navigate thru the UI and check the colors for each resource type.

## Test Impact
No automated test impact as this is only a change in colors.

## Screen shots

image

image

image

image

image

image

## Request review criteria:
Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)
- [x] Included any necessary screenshots or gifs if it was a UI change.
- [x] Included tags to the UX team if it was a UI/UX change.

/cc @simrandhaliw @tobiastal

### CODE REVIEW

**Comment by simrandhaliw** [comment]
The changes listed above LGTM! One item that might be missing is the Projects icon background colour. 

@tobiastal have we finalized whether the Projects page icon will have a grey background or not?

**Comment by tobiastal** [comment]
Yes, the Projects page icon should have a grey background.
It's listed in the new [guidelines](https://docs.google.com/document/d/1OdaE6LjTikTMTIOHvgGQzRDJpm9Bn-kPxsfoqRh5tsU/edit#heading=h.v94u92bpa671)

**Comment by jeff-phillips-18** [comment]
@simrandhaliw @tobiastal I have made the change and updated the image above.

**Comment by jenny-s51** [comment]
Please kindly hold from merging/approving this PR - currently reviewing these updates.

**Review by jenny-s51 — COMMENTED** [review]
Looks great @jeff-phillips-18 ! Navigated through the UI and checked the colors for each resource type, updates look and work as expected. 

Just a quick question regarding the "hovered/clicked" vs "default" border states. For the cases where cards are hovered/clicked ie: 
Screenshot 2024-10-29 at 10 41 50 AM

Wondering if these should apply the border hover / clicked color? Currently they're using the border default color when these cards are hovered or selected.

**Comment by tobiastal** [comment]
> Looks great @jeff-phillips-18 ! Navigated through the UI and checked the colors for each resource type, updates look and work as expected.
> 
> Just a quick question regarding the "hovered/clicked" vs "default" border states. For the cases where cards are hovered/clicked ie: Screenshot 2024-10-29 at 10 41 50 AM
> 
> Wondering if these should apply the border hover / clicked color? Currently they're using the border default color.

Yes, you're correct. The default border color should be used for items with a border that aren’t hovered or clicked (e.g., cards with a static border color, though that’s not the case here). The "hovered/clicked" colors should be used for this situation.

**Comment by jeff-phillips-18** [comment]
@jenny-s51 @tobiastal 

Updated to use selected color:

image

File: frontend/src/concepts/design/vars.scss [hunk 1]
```diff
@@ -1,48 +1,97 @@
 :root {
   --ai-set-up--BackgroundColor: #ffe8cc;
-  --ai-organize--BackgroundColor: #ffe8cc;
-  --ai-training--BackgroundColor: #daf2f2;
-  --ai-serving--BackgroundColor: #e0f0ff;
-
+  --ai-set-up--LabelBackgroundColor: #fccb8f;
   --ai-set-up--BorderColor: #f8ae54;
-  --ai-organize--BorderColor: #f8ae54;
+  --ai-set-up--SelectedColor: #f8ae54;
```
Comment (jenny-s51) [inline frontend/src/concepts/design/vars.scss:5]:
```suggestion
  --ai-set-up--SelectedColor: #f5921b;
```

**Review by jenny-s51 — COMMENTED** [review]
Just the small fix above, otherwise LGTM! Thanks for making these updates @jeff-phillips-18

**Comment by jeff-phillips-18** [comment]
> Just the small fix above, otherwise LGTM! Thanks for making these updates @jeff-phillips-18

Thanks @jenny-s51 I made that change.

**Review by tobiastal — APPROVED** [review]

**Review by jenny-s51 — APPROVED** [review]

**Pull request merged**